### PR TITLE
Updated Rooms API Specification Per Internal Review

### PR DESCRIPTION
**Pull request by mikehang-msft**
# Data Plane API - Pull Request

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).

* Link to engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [ ] a public preview
- [ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>
* Design Document:
* Previous Open API Doc:
* Updated paths:

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azapi) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azapi).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

**Review by Mrayyan — APPROVED** [review]
I guess you can delete the file we had for '23 - 01 - 01

File: specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+      "type": "object",
+      "properties": {
+        "title": {
+          "description": "The room title.",
+          "type": "string",
+          "example": "Meeting with Alice and Bob"
+        },
+        "validFrom": {
+          "format": "date-time",
+          "description": "The timestamp from when the room is open for joining. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. If this value is not set, its value is set to a current date time by default.",
```
Comment (minnieliu) [inline specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json:353]: nit: consider making description more succinct: 
"The timestamp from when the room is open for joining. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. The default value is the current date time."

File: specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+          "type": "string",
+          "example": "2021-09-07T07:55:41Z"
+        },
+        "validUntil": {
+          "format": "date-time",
+          "description": "The timestamp from when the room can no longer be joined. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. If this value is not set, its value is set to a current date time plus 180 days by default.",
```
Comment (minnieliu) [inline specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json:359]: nit: consider making description more succinct: 
"The timestamp from when the room can no longer be joined. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. The default value is the current date time plus 180 days."

**Review by minnieliu — APPROVED** [review]

File: specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json (hunk 1 above)
Comment (Mrayyan) [reply specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json:353]: +1 ^

File: specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json (hunk 2 above)
Comment (mikehang-msft) [reply specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json]: resolved Minnie

File: specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json (hunk 1 above)
Comment (mikehang-msft) [reply specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json]: resolved Minnie

**Pull request merged**